extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -130,8 +130,8 @@ export class BookTocManager implements IBookTocManager {
 		while (await fs.pathExists(path.join(newFileName.concat(' - ', counter.toString())).concat(path.parse(dest).ext))) {
 			counter++;
 		}
-		await fs.move(src, path.join(newFileName.concat(' - ', counter.toString())).concat(path.parse(dest).ext), { overwrite: true });
 		this.movedFiles.set(src, path.join(newFileName.concat(' - ', counter.toString())).concat(path.parse(dest).ext));
+		await fs.move(src, path.join(newFileName.concat(' - ', counter.toString())).concat(path.parse(dest).ext), { overwrite: true });
```

Comment:
Would be nice to create a var for the full path so we aren't having to concat it a bunch of duplicate times (130, 133, 134, 135, 136)

Same comment for the other places you changed that have similar duplication